packages/test/mocha-test-setup/package.json

```diff
@@ -75,8 +110,9 @@
 				"script": false
 			},
 			"tsc": [
-				"...",
-				"typetests:gen"
+				"typetests:gen",
+				"build:genver",
+				"^tsc"
 			]
```

Comment:
Looks wrong. This doesn't have typetests, don't know what that is here for. In general, I think ... is preferred. The policy messages can be a little misleading. And if tsc needs build:genver, then I would expect build:esnext to as well.
@tylerbutler, why does this seem so wrong to me?

Reply:
did a small test by removing the fluidBuild config. the policy check states mocha-test-setup as a public package: `Policy violations for public package "@fluid-internal/mocha-test-setup":`

Reply:
removed the typetests:gen scripts and the `tsc` fluidBuild config